### PR TITLE
resolved date issue and added seasonal stats for players and opponent…

### PR DESCRIPTION
**Pull request by RBecker262**
Quite an involved set of changes with two focuses...
1. fix the date issue for "today" on the three master file builds (schedule, player, team)
2. added relevant seasonal stats to player
3. added opponent name and time of game info to team master

Opponent will read like "at Chicago Cubs" for the Cardinals team entry and "vs St Louis Cardinals" for the Cubs team entry. Game time is in local time based on team. Also did a kick ass job on comments and keeping functions small since I added a LOT of code.

### CODE REVIEW

File: updplayer.py [hunk 1]
```diff
@@ -157,13 +165,164 @@ def process_schedule(sched_dict):
     return daily_player_dict
 
 
-def search_dictionary(indict, hometeam, awayteam, teamcode, resultdict):
+def build_pitching_stats(keylist, indict):
+    """
+    :param keylist: list of keys in indict dictionary
+    :param indict:  boxscore dictionary from which to extract player data
```
Comment (alesage) [inline updplayer.py]: I would just extract the keys for the client e.g. using indict.keys() unless there's a reason these might differ?

File: updplayer.py [hunk 2]
```diff
@@ -157,13 +165,164 @@ def process_schedule(sched_dict):
...
+                    'ip': ip,
+                    'saves': saves}
+
+    return (statkey, pitcherstats)
+
+
+def build_batting_stats(keylist, indict):
+    """
+    :param keylist: list of keys in indict dictionary
+    :param indict:  boxscore dictionary from which to extract player data
```
Comment (alesage) [inline updplayer.py]: Ditto above.

**Review by alesage — CHANGES_REQUESTED** [review]
I like it, sorry for the delay in reviewing--in my defense this actually is a large review, I'd suggest (next time) doing the date changes, e.g. as a separate pull request b/c that's a no-brainer and quick to press 'yes' on.

Any thoughts on that one signature change I suggested?

File: updplayer.py (hunk 1 above)
Comment (RBecker262) [reply updplayer.py]: Agreed, made change, tested, and pushed up here. Just looked at notes and forgot about putting the param to close an issue, will try to remember next time.

File: updplayer.py (hunk 2 above)
Comment (RBecker262) [reply updplayer.py]: Ditto my comment above.

**Review by alesage — APPROVED** [review]

**Pull request merged**